_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -59,6 +73,30 @@ class HealthCareApp extends React.Component {
     this.state = initialState;
   }
 
+  getNextUrl() {
+    const routes = this.props.route.childRoutes;
+    const panels = [];
+    let currentPath = this.props.location.pathname;
+    let nextPath = '';
+
+    if (currentPath === '/') {
```

Comment:
Can you add a TODO assigned to me to "remove the `/` alias for `/introduction` using history.replaceState()"